index.html

```diff
@@ -48,6 +48,109 @@ <h1 class="logo-text">Pascal</h1>
         </div>
     </section>
 
+    <section class="work">
+
+        <div class="card">
+            <div class="card-header">
+            </div>
+            <h2 class="card-titles">TONIC</h2>
+
+
+            <ul class="bandle">
+                <li>
+                    <p class="canopy">CANOPY</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">Backend Dev</p>
+                </li>
+                <li><img src="./img/cannopy.png" class="canopy-circle" alt="canopy" /></li>
+                <li>
+                    <p class="role">2015</p>
+                </li>
+            </ul>
+
+            <div class="card-content">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="content_btn">
+                <li class="data for-html">
+                    <p>html</p>
+                </li>
+                <li class="data for-css">
+                    <p>CSS</p>
+                </li>
+                <li class="data for-js">
+                    <p>JavaSript</p>
+                </li>
+            </ul>
+            <div class="see-project">
+                <a href="#" class="see">See Project</a>
+            </div>
+
+        </div>
+
+
+    </section>
+    <section class="about">
```

Comment:
## OPTIONAL
- [ ] Please this section has the proper background image (decorations are always backgrounded images, not `img` tags) 👍 

- [ ] Original Design:
![y2](https://user-images.githubusercontent.com/63972670/159093418-ead4ef6e-fd40-405b-a535-ff3d15ac550c.JPG)

- [ ] Your Design: 
![f3](https://user-images.githubusercontent.com/63972670/159093666-71ba8111-4906-4855-88c2-3ae069ad3557.JPG)

Reply:
Done, thank you!